src/transformers/models/roformer/modeling_roformer.py

```diff
@@ -262,6 +262,15 @@ def forward(
         elif past_key_value is not None:
             key_layer = self.transpose_for_scores(self.key(hidden_states))
             value_layer = self.transpose_for_scores(self.value(hidden_states))
+            if sinusoidal_pos is not None:
+                if self.rotary_value:
+                    query_layer, key_layer, value_layer = self.apply_rotary_position_embeddings(
+                        sinusoidal_pos, query_layer, key_layer, value_layer
+                    )
+                else:
+                    query_layer, key_layer = self.apply_rotary_position_embeddings(
+                        sinusoidal_pos, query_layer, key_layer
+                    )
```

Comment:
This is done twice (here and in the next else. This is a small nits!

Reply:
Thank you for your carefulness, this is a great first PR experience for me.